src/view/UserInterface.py

```diff
@@ -73,3 +73,7 @@ def showMessage(self):
     @abstractmethod
     def showCorrectGuess(self):
         pass
+    

```

Comment:
Did you remove exit?

Reply:
Needs merge conflict resolved.  I'm unsure if you still plan to use showExit or not.